test/cairo_compliance_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule NifTest do
```

Comment:
mix test error: cannot define module NifTest because it is currently being defined in test/cairo_test.exs:1
The name `NifTest` seems to be used in both cairo_test.exs and cairo_compliance_test.exs.